src/test/kotlin/com/ampnet/payoutservice/util/json/MerkleTreeJsonSerializerTest.kt

```diff
@@ -0,0 +1,131 @@
+package com.ampnet.payoutservice.util.json
+
+import com.ampnet.payoutservice.TestBase
+import com.ampnet.payoutservice.util.AccountBalance
+import com.ampnet.payoutservice.util.Balance
+import com.ampnet.payoutservice.util.HashFunction
+import com.ampnet.payoutservice.util.MerkleTree
+import com.ampnet.payoutservice.util.MerkleTree.Companion.NilNode
+import com.ampnet.payoutservice.util.WalletAddress
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import java.math.BigInteger
+
+class MerkleTreeJsonSerializerTest : TestBase() {
+
+    private val objectMapper = jacksonObjectMapper()
+
+    @Test
+    fun mustCorrectlySerializeSimpleMerkleTree() {
+        val accountBalance = AccountBalance(WalletAddress("0x0"), Balance(BigInteger("0")))
+        val tree = suppose("simple Merkle tree is created") {
+            MerkleTree(
+                listOf(accountBalance),
+                HashFunction.IDENTITY
+            )
+        }
+
+        val serializedTree = suppose("simple Merkle tree is serialized to JSON") {
+            objectMapper.valueToTree<JsonNode>(tree)
+        }
+
+        verify("simple Merkle tree is correctly serialized") {
+            Assertions.assertThat(serializedTree).withMessage().isEqualTo(
```

Comment:
Use just `assertThat` like in other tests.
```suggestion
            assertThat(serializedTree).withMessage().isEqualTo(
```